cdap-ui/server/express.js

```diff
@@ -60,6 +61,42 @@ function makeApp (authAddress, cdapConfig) {
               '.constant("MY_CONFIG",'+data+');');
   });
 
+  app.post('/downloadQuery', function(req, res) {
+    var fs = require('fs');
+    var StringDecoder = require('string_decoder').StringDecoder;
+    var url = req.body.backendUrl;
+    var query = req.body.queryHandle;
+
+    var path = DIST_PATH + '/assets/public';
+
+    fs.mkdir(path, function (err) {
+      if (err && err.code === 'EEXIST') {
```

Comment:
doesn't seem like we need an error handling callback at all?
(since it's just a no-op)